contracts/src/v0.8/automation/dev/v2_3/AutomationRegistryLogicB2_3.sol

```diff
@@ -458,8 +467,8 @@ contract AutomationRegistryLogicB2_3 is AutomationRegistryBase2_3 {
   {
     state = State({
       nonce: s_storage.nonce,
-      ownerLinkBalance: 0,
-      expectedLinkBalance: s_reserveAmounts[address(i_link)],
+      ownerLinkBalance: 0, // deprecated
```

Comment:
thank you for making efforts to make this backwards compatible!

Reply:
ofc! it's a team win :)